src/components/CCPDashboard/PatientInfoTable.tsx

```diff
@@ -212,137 +230,291 @@ export const PatientInfoTable = ({
+          className={classes.tableCell}
+          style={{
+            width: '78px',
+            maxWidth: '78px',
+            borderLeft: `0px solid ${triageLevels[patient.triageLevel].colour}`,
+            color: Colours.InactiveGrey,
+          }}
+        >
+          {triageLevels[patient.triageLevel].label}
+        </TableCell>
+        <TableCell
+          className={classes.tableCell}
+          width="94px"
+          style={{ maxWidth: '94px', color: Colours.InactiveGrey }}
+        >
+          {patient.barcodeValue}
+        </TableCell>
+        <TableCell
+          className={classes.tableCell}
+          width="72px"
+          style={{ maxWidth: '72px', color: Colours.InactiveGrey }}
+        >
+          {patient.gender}
+        </TableCell>
+        <TableCell
+          className={classes.tableCell}
+          width="34px"
+          style={{ maxWidth: '34px', color: Colours.InactiveGrey }}
+        >
+          {patient.age}
+        </TableCell>
+        <TableCell
+          className={classes.tableCell}
+          width="104px"
+          style={{ maxWidth: '104px', color: Colours.InactiveGrey }}
+        >
+          {statusLabels[patient.status]}
+        </TableCell>
+        {type === CCPDashboardTabOptions.PatientOverview && (
+          <>
+            <TableCell
+              className={classes.tableCell}
+              width="128px"
+              style={{ maxWidth: '128px', color: Colours.InactiveGrey }}
             >
-              <MoreHoriz />
-            </IconButton>
-          </TableCell>
-        </TableRow>
-      );
-    }
-  );
+              {patient.hospitalId?.name}
+            </TableCell>
+            <TableCell
+              className={classes.tableCell}
+              width="192px"
+              style={{ maxWidth: '192px', color: Colours.InactiveGrey }}
+            >
+              {moment(patient.updatedAt).format('MMM D YYYY, h:mm A')}
+            </TableCell>
+          </>
+        )}
+        {type === CCPDashboardTabOptions.Hospital && (
+          <>
+            <TableCell
+              className={classes.tableCell}
+              width="132px"
+              style={{ maxWidth: '132px', color: Colours.InactiveGrey }}
+            >
+              {patient.runNumber}
+            </TableCell>
+            <TableCell
+              className={classes.tableCell}
+              width="192px"
+              style={{ maxWidth: '192px', color: Colours.InactiveGrey }}
+            >
+              {moment(patient.transportTime).format('MMM D YYYY, h:mm A')}
+            </TableCell>
+          </>
+        )}
+        <TableCell
+          width="48px"
+          style={{ maxWidth: '48px', color: Colours.InactiveGrey }}
+        >
+          <IconButton
+            color="inherit"
+            onClick={(e) => handleClickRestoreMenu(e, patient)}
+          >
+            <MoreHoriz />
+          </IconButton>
+        </TableCell>
+      </TableRow>
+    );
+  });
```

Comment:
This could be a more generalized component that's used twice where you can use a different colour depending on whether you pass deleted or active patients in.

Reply:
yes please - there's a lot of extraneous repeated code which i'm not a fan of

Reply:
yeah I agree